src/model.js

```diff
@@ -155,7 +155,11 @@ ngFileUpload.service('Upload', ['$parse', '$timeout', '$compile', '$q', 'UploadE
       var invalidModel = upload.attrGetter('ngfModelInvalid', attr);
       if (invalidModel) {
         $timeout(function () {
-          $parse(invalidModel).assign(scope, isSingleModel ? invalidFile : invalidFiles);
+          if (invalidModel.includes('(')){
```

Comment:
This makes it seem like you have to pass the function in like
```
ngf-model-invalid="method()"
```
rather than
```
ngf-model-invalid="method"
```

Perhaps parse it and use typeof to check instead?